jdaviz/configs/imviz/tests/test_parser.py

```diff
@@ -270,6 +270,7 @@ def test_parse_jwst_nircam_level2(self, imviz_helper):
         phot_plugin.counts_factor = (data.meta['photometry']['conversion_megajanskys'] /
                                      data.meta['exposure']['exposure_time'])
         assert_allclose(phot_plugin.counts_factor, 0.0036385915646798953)
+        assert_allclose(phot_plugin.flux_scaling, 3.631e-3)
         phot_plugin.flux_scaling = 1  # Simple mag, no zeropoint
```

Comment:
Doh... I missed this part. You have to remove this line. And update the expected mag answer to match your AB mag prediction.
```suggestion
```

Reply:
If the aperture sum is expected to be 4.486487e-11 MJy , what is its AB magnitude? 😉

Reply:
Updated the tests!